ow_lander/scripts/trajectory_publisher.py

```diff
@@ -131,24 +133,50 @@ def talker(req):
       print "Sent %s on joint[0] publisher"%(float("%14s\n"%row[12+0]))
       rate.sleep()
 
-        
+
   if guarded_move_bool : # If the activity is a guarded_move
+
+    # Start publisher for guarded_move result message
+    pubs.append(rospy.Publisher('/GUARDED_MOVE_RESULT_W0OOOOHHHHHHHH', GuardedMoveResult, queue_size=10))
```

Comment:
Yes, please change this name to something not ridiculous. In general, it's better to make all the changes you know you need to make before starting the code review.

Reply:
Yes, I am changing it, I just wanted to start the review soon, before our next meeting, in case you had constructive feedback to share during the meeting